### PR TITLE
fix: update make targets in cookiecutters templates

### PR DESCRIPTION
**Pull request by UsamaSadiq**
**Issue:** [BOM-2134](https://openedx.atlassian.net/browse/BOM-2134)

### Description
- Updated the existing make targets and ensured all required targets' presence in cookie-cutters templates.

### CODE REVIEW

File: cookiecutter-xblock/{{cookiecutter.repo_name}}/tox.ini [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+; D101 = Missing docstring in public class
+; D200 = One-line docstring should fit on one line with quotes
+; D203 = 1 blank line required before class docstring
+; D212 = Multi-line docstring summary should start at the first line
+; D215 = Section underline is over-indented (numpy style)
+; D404 = First word of the docstring should not be This (numpy style)
+; D405 = Section name should be properly capitalized (numpy style)
+; D406 = Section name should end with a newline (numpy style)
+; D407 = Missing dashed underline after section (numpy style)
+; D408 = Section underline should be in the line following the section’s name (numpy style)
```
Comment (Jawayria) [inline cookiecutter-xblock/{{cookiecutter.repo_name}}/tox.ini:23]: This apostrophe in `section's` is a non-ascii character. Please use simple apostrophe from the keyboard as we've recently removed non-ascii characters from current repos.

Comment (UsamaSadiq) [reply cookiecutter-xblock/{{cookiecutter.repo_name}}/tox.ini:23]: Didn't we add any new check to avoid adding any such values in future?

Comment (UsamaSadiq) [reply cookiecutter-xblock/{{cookiecutter.repo_name}}/tox.ini]: Removed all non-ascii characters similar to the one pointed out above.

**Review by Jawayria — APPROVED** [review]

File: cookiecutter-django-ida/{{cookiecutter.repo_name}}/Makefile [hunk 2]
```diff
@@ -71,16 +72,8 @@ coverage: clean
...
-	isort --recursive --atomic {{cookiecutter.project_name}}/
-
-style: ## run Python style checker
-	pylint --rcfile=pylintrc {{cookiecutter.project_name}} *.py
-
-lint: ## run Python code linting
-	pylint --rcfile=pylintrc {{cookiecutter.project_name}} *.py
-
-quality: style isort_check lint ## check code style and import sorting, then lint
+quality:
```
Comment (aht007) [inline cookiecutter-django-ida/{{cookiecutter.repo_name}}/Makefile:75]: What I am getting from all these removed commands and new `quality` target is that all these commands will be executed within the `tox` command but what if we still want to run let say just lint command then I wouldn't have any make target for that. We can still keep the old make targets intact in the file and just update the quality check to run tox.

Comment (UsamaSadiq) [reply cookiecutter-django-ida/{{cookiecutter.repo_name}}/Makefile:84]: updated the targets as per the suggestion.

**Review by aht007 — APPROVED** [review]

**Pull request merged**